loopr-ios.xcodeproj/project.pbxproj

```diff
@@ -2889,11 +2889,11 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 47TZG75K6F;
```

Comment:
These changes in project.pbxproj should not be in the commit. 47TZG75K6F is for your Apple ID. The app is using org.loopring.alpha.loopr-ios now.

Reply:
updated